app/views/alchemy/admin/partials/_upload_button.html.erb

```diff
@@ -0,0 +1,31 @@
+<%= form_for [alchemy, :admin, form_object], html: {multipart: true, class: 'upload-button'} do |f| %>
```

Comment:
If we pass the `alchemy` url proxy here, the button is not usable outside of Alchemy